src/commands/mfe/add.ts

```diff
@@ -33,9 +46,15 @@ export default class Add extends Command {
 
     const { args, flags } = await this.parse(Add)
 
+    if (flags.slot && flags.type !== MicroFrontendType.AppBuilder) {
```

Comment:
Use dependsOn instead https://oclif.io/docs/flags

Reply:
`dependsOn` basically just checks if the other flag exists, not allowing to check for specific value. That's actually an [open issue](https://github.com/oclif/oclif/issues/323) in oclif.